packages/rke2-calico/generated-changes/patch/values.yaml.patch

```diff
@@ -56,7 +56,5 @@
 +  featureDetectOverride: "ChecksumOffloadBroken=true"
 +  healthPort: 9099
 +  defaultEndpointToHostAction: "Drop"
-+  failsafeInboundHostPorts: ""
-+  failsafeOutboundHostPorts: ""
```

Comment:
If you remove these two, they will not be able to be set via helm

Reply:
Yes you can with the new logic you can specify whatever value you want through helm.

Reply:
ok, got it, thanks. The problem is that in Rancher UI, users will not be able to see what parameters can be changed, only the ones in values. So it is good if we keep the most common ones